features/extension_activation.feature

```diff
@@ -9,9 +9,8 @@ Feature: Test extension/module activation
   # once it does.
   @skip-sles-15
   Scenario: Lists all possible extensions
-    When I run `SUSEConnect --list-extensions`
-    Then the exit status should be 0
```

Comment:
I mean, you've removed it here. Asking about the other check for consistency.

Reply:
Using `I successfully run` will automatically check that the exit status is 0, which is implied by "successfully"... Not sure if that answers your question.